[release-24.11] zed-editor: 0.161.2 -> 0.173.10

GaetanLepage opened the pull request:
## Things done

Attempt at updating `zed-editor` on the stable channel.

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

qbisi committed:
netgen: init at 6.2.2501

Raroh73 committed:
vscode-extensions.continue.continue: 0.8.54 -> 0.8.68

Raroh73 committed:
vscode-extensions.astro-build.astro-vscode: 2.10.2 -> 2.15.4

Fixed hash

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

Raroh73 committed:
vscode-extensions.davidanson.vscode-markdownlint: 0.56.0 -> 0.58.2

Raroh73 committed:
vscode-extensions.editorconfig.editorconfig: 0.16.4 -> 0.17.0

Raroh73 committed:
vscode-extensions.redhat.vscode-yaml: 1.14.0 -> 1.16.0

toastal committed:
nixos/movim: run nixfmt on module

This 711-line file was expanded into 817-line file by nixfmt.
Readability was hurt as now I can’t see as much in my editor at a time;
this directly makes editing & reviewing slower as reading is harder. I
am upset about this change.

vbgl committed:
jasmin-compiler: 2024.07.3 → 2025.02.0

ejstrunz committed:
linux_latest-libre: 19712 -> 19729

kernel version 6.6.80 fails to compile without this patch

NickCao committed:
opensbi: 1.5.1 -> 1.6

Diff: https://github.com/riscv-software-src/opensbi/compare/v1.5.1...v1.6

qbisi committed:
superlu_dist: init at 9.1.0

bbenno committed:
vscode-extensions.castwide.solargraph: init at 0.24.1

r-ryantm committed:
fzf: 0.60.2 -> 0.60.3

brainrake commented:
Zed was updated to 175.5 here
https://github.com/NixOS/nixpkgs/commit/e5c850779dad4b97d52678d45c7b256973b28c8d
So I'll go ahead and close this.

brainrake commented:
Sorry, I didn't notice this was on the release branch.

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.clang: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.lld: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.libunwind: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.openmp: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.libcxx: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.compiler-rt: move patches out of common

RossComputerGuy committed:
llvmPackages_{19,20,git}.bolt: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.lldb: move patches out of common

RossComputerGuy committed:
llvmPackages_{12,13,14,15,16,17,18,19,20,git}.llvm: move patches out of common

Stunkymonkey committed:
youtube-tui: use fetchCargoVendor

Stunkymonkey committed:
youtube-tui: 0.8.0 -> 0.8.1

Diff: https://github.com/Siriusmart/youtube-tui/compare/v0.8.0...v0.8.1

r-ryantm committed:
fcft: 3.1.10 -> 3.2.0

r-ryantm committed:
gh: 2.67.0 -> 2.68.1

rhelmot committed:
libinotify-kqueue: 20180201 -> 20240724

siriobalmelli committed:
aider-chat: add withPlaywright override

Signed-off-by: Sirio Balmelli <sirio@b-ad.ch>

siriobalmelli committed:
python3Packages.aider-chat: 0.75.1 -> 0.75.2

Signed-off-by: Sirio Balmelli <sirio@b-ad.ch>

toastal committed:
.git-blame-ignore-revs: add nixos/movim

r-ryantm committed:
papeer: 0.8.3 -> 0.8.4

Commit:
vscode-extension.johnpapa.winteriscoming: init at 1.4.4

NickCao committed:
routinator: 0.14.1 -> 0.14.2 (#387105)

NickCao committed:
wasmtime: 30.0.0 -> 30.0.2 (#387196)

NickCao committed:
libretro.parallel-n64: 0-unstable-2025-02-06 -> 0-unstable-2025-03-02 (#387195)

NickCao committed:
libretro.nestopia: 0-unstable-2025-01-19 -> 0-unstable-2025-02-22 (#387191)

r-ryantm committed:
python312Packages.commitizen: 4.2.2 -> 4.4.1

pbsds committed:
agorakit: avoid using finalAttrs.pname

pbsds committed:
base45: avoid using finalAttrs.pname

pbsds committed:
chsrc: avoid using finalAttrs.pname

pbsds committed:
iio-oscilloscope: avoid using finalAttrs.pname

pbsds committed:
frr: avoid using finalAttrs.pname

pbsds committed:
misskey: avoid using finalAttrs.pname

pbsds committed:
minutor: avoid using finalAttrs.pname

pbsds committed:
tart: avoid using finalAttrs.pname

pbsds committed:
ytmdesktop: avoid using finalAttrs.pname

pbsds committed:
tslib: avoid using finalAttrs.pname

pbsds committed:
zinit: avoid using finalAttrs.pname

NickCao committed:
youki: 0.5.1 -> 0.5.2 (#387281)

pbsds committed:
stellarsolver: avoid using finalAttrs.pname

NickCao committed:
rcu: 2025.001r -> 2025.001s (#387261)

NickCao committed:
ssdfs-utils: 4.49 -> 4.50 (#387260)

NickCao committed:
backblaze-b2: 4.3.0 -> 4.3.1 (#387257)

NickCao committed:
webex: 44.10.2.31237 -> 45.2.0.31846 (#387229)

NickCao committed:
python313Packages.aiogithubapi: relax async-timeout (#387227)

NickCao committed:
kdash: 0.6.1 -> 0.6.2 (#387222)

NickCao committed:
libretro.bsnes: 0-unstable-2025-01-17 -> 0-unstable-2025-02-28 (#387206)

NickCao committed:
buildkit: 0.20.0 -> 0.20.1 (#387358)

drupol committed:
protonmail-bridge: 3.17.0 -> 3.18.0 (#387714)

NickCao committed:
python312Packages.edk2-pytool-library: 0.22.5 -> 0.22.6 (#387353)

NickCao committed:
jbang: 0.123.0 -> 0.124.0 (#387332)

drupol committed:
python312Packages.debugpy: 1.8.12 -> 1.8.13 (#387707)

drupol committed:
mergiraf: 0.5.1 -> 0.6.0 (#387724)

NickCao committed:
python312Packages.pdm-build-locked: 0.3.4 -> 0.3.5 (#387402)

NickCao committed:
terraform-providers.ibm: 1.75.2 -> 1.76.0 (#387380)

drupol committed:
treewide: avoid using finalAttrs.pname (#387725)

drupol committed:
python312Packages.commitizen: 4.2.2 -> 4.4.1 (#387728)

kira-bruneau committed:
ukmm: use fetchCargoVendor (#387691)

NickCao committed:
rpl: 1.17 -> 1.18 (#387456)

NickCao committed:
licensure: 0.7.1 -> 0.7.2 (#387436)

NickCao committed:
jumpy: use fetchCargoVendor (#387431)

NickCao committed:
goxlr-utility: use fetchCargoVendor (#387427)

NickCao committed:
hyperspeedcube: use fetchCargoVendor & 1.0.7 -> 1.0.12 (#387424)

NickCao committed:
aliyun-cli: 3.0.254 -> 3.0.256 (#387509)

pbsds committed:
yek: 0.16.0 -> 0.21.0 (#380294)

NickCao committed:
moar: 1.31.3 -> 1.31.4 (#387485)

sikmir committed:
glaze: 4.3.1 -> 5.0.0 (#379791)

NickCao committed:
step-cli: 0.28.3 -> 0.28.5 (#387484)

NickCao committed:
plantuml-server: 1.2025.0 -> 1.2025.2 (#387477)

NickCao committed:
python313Packages.pykulersky: 0.5.5 -> 0.5.8 (#387563)

NickCao committed:
terraform-providers.okta: 4.14.0 -> 4.14.1 (#387561)

NickCao committed:
python313Packages.aiolifx: 1.1.3 -> 1.1.4 (#387559)

NickCao committed:
solo5: 0.9.0 -> 0.9.1 (#387544)

NickCao committed:
python313Packages.publicsuffixlist: 1.0.2.20250302 -> 1.0.2.20250306 (#387529)

NickCao committed:
pgmetrics: 1.17.0 -> 1.17.1 (#387527)

NickCao committed:
jsonfmt: 0.5.0 -> 0.5.1 (#387521)

NickCao committed:
hermitcli: 0.44.0 -> 0.44.1 (#387519)

drupol committed:
zed-editor: switch to `finalAttrs` pattern, minor cleanup

drupol committed:
zed-editor: 0.176.1 -> 0.176.2

NickCao committed:
goflow2: 2.2.1 -> 2.2.2 (#387518)

NickCao committed:
pywal16: 3.8.0 -> 3.8.2 (#387598)

NickCao committed:
flannel: 0.26.4 -> 0.26.5 (#387595)

NickCao committed:
bundletool: 1.18.0 -> 1.18.1 (#387589)

NickCao committed:
python313Packages.pycfmodel: 1.0.1 -> 1.1.0 (#387565)

r-ryantm committed:
vscode-extensions.ms-python.vscode-pylance: 2025.2.1 -> 2025.3.1

r-ryantm committed:
openapi-changes: 0.0.75 -> 0.0.76

NickCao committed:
agate: 3.3.12 -> 3.3.13 (#387635)

NickCao committed:
starlark: 0-unstable-2025-02-05 -> 0-unstable-2025-02-25 (#387623)

NickCao committed:
subxt: 0.39.0 -> 0.40.0 (#387622)

NickCao committed:
skate: 1.0.0 -> 1.0.1 (#387621)

NickCao committed:
terraform-providers.digitalocean: 2.49.0 -> 2.49.1 (#387672)

NickCao committed:
picom-pijulius: 8.2-unstable-2025-02-19 -> 8.2-unstable-2025-03-01 (#387660)

NickCao committed:
kubergrunt: 0.16.0 -> 0.17.0 (#387702)

NickCao committed:
gvproxy: 0.8.3 -> 0.8.4 (#387695)

NickCao committed:
json-sort-cli: 2.0.2 -> 2.0.3 (#387693)

NickCao committed:
terraform-providers.snowflake: 1.0.3 -> 1.0.4 (#387679)

drupol committed:
vscode-extensions.ms-python.vscode-pylance: 2025.2.1 -> 2025.3.1 (#387737)

Stunkymonkey committed:
youtube-tui: use fetchCargoVendor & 0.8.0 -> 0.8.1 (#387438)

NickCao committed:
chirp: 0.4.0-unstable-2025-02-20 -> 0.4.0-unstable-2025-03-05 (#387677)

NickCao committed:
eigenmath: 337-unstable-2025-02-18 -> 337-unstable-2025-03-05 (#387674)

drupol committed:
auth0-cli: 1.9.1 -> 1.9.2 (#387717)

NickCao committed:
nixos/dendrite: rename settings.sync_api.search.enable option to sett… (#386393)

4evy committed:
vscode-extensions.github.copilot: 1.275.0 -> 1.277.1411

4evy committed:
vscode-extensions.github.copilot-chat: 0.24.2025021302 -> 0.25.2025021201

kira-bruneau committed:
yabridgectl: use fetchCargoVendor (#387696)

r-ryantm committed:
python312Packages.mplhep: 0.3.56 -> 0.3.57

happysalada committed:
{lnx,lsp-ai,lapce,lazymc,lightdm-mobile-greeter}: use fetchCargoVendor (#387583)

happysalada committed:
cargo-tauri: 2.2.7 -> 2.3.1 (#387659)

GaetanLepage committed:
mujoco: 3.2.7 -> 3.3.0

Diff:
https://github.com/google-deepmind/mujoco/compare/refs/tags/3.2.7...3.3.0

Changelog: https://mujoco.readthedocs.io/en/3.3.0/changelog.html

rhelmot committed:
vips: Fix cross build for FreeBSD (#387409)

GaetanLepage committed:
python312Packages.cf-xarray: 0.10.0 -> 0.10.1

Diff: https://github.com/xarray-contrib/cf-xarray/compare/refs/tags/v0.10.0...v0.10.1

Changelog: https://github.com/xarray-contrib/cf-xarray/releases/tag/v0.10.1

rycee committed:
svtplay-dl: 4.101 -> 4.103

r-ryantm committed:
beam26Packages.elixir-ls: 0.26.4 -> 0.27.1

r-ryantm committed:
beam26Packages.ex_doc: 0.37.2 -> 0.37.3

GaetanLepage committed:
zed-editor: 0.176.1 -> 0.176.2 (#387735)

nixpkgs-merge-bot[bot] committed:
openapi-changes: 0.0.75 -> 0.0.76 (#387738)

Co-authored-by: mguentner <mguentner@users.noreply.github.com>

adamcstephens committed:
beam26Packages.elixir-ls: 0.26.4 -> 0.27.1 (#387758)

NickCao committed:
python312Packages.cf-xarray: 0.10.0 -> 0.10.1 (#387754)

NickCao committed:
mujoco: 3.2.7 -> 3.3.0 (#387752)

r-ryantm committed:
clever-tools: 3.11.0 -> 3.12.0

r-ryantm committed:
libdnf: 0.73.4 -> 0.74.0

r-ryantm committed:
python312Packages.google-genai: 1.4.0 -> 1.5.0

r-ryantm committed:
python312Packages.gitingest: 0.1.3 -> 0.1.4

r-ryantm committed:
python312Packages.mandown: 1.11.1 -> 1.11.2

adamcstephens committed:
woodpecker-{server,agent,cli} 3.1.0 -> 3.3.0 (#387703)

adamcstephens committed:
beamPackages.ex_doc: add erlang builds to tests

dotlambda committed:
python312Packages.google-genai: 1.4.0 -> 1.5.0 (#387780)

drupol committed:
vscode-extensions.castwide.solargraph: init at 0.24.1 (#387656)

drupol committed:
python312Packages.gitingest: 0.1.3 -> 0.1.4 (#387781)

nixpkgs-merge-bot[bot] committed:
globalping-cli: 1.4.4 -> 1.5.0 (#387653)

Co-authored-by: XYenon <XYenon@users.noreply.github.com>

Emin017 committed:
vyxal: update mill dependency to use fetchurl for version and URL locking

The source link for mill will be changing, so we need to specify the
download link for mill here.

Signed-off-by: Qiming Chu <cchuqiming@gmail.com>

adamcstephens committed:
beam26Packages.ex_doc: 0.37.2 -> 0.37.3 (#387767)

RossComputerGuy committed:
zig: 0.13 -> 0.14 (#387154)

r-ryantm committed:
wit-bindgen: 0.39.0 -> 0.40.0

adamcstephens committed:
localstack: fix build, move to toplevel, 4.0.3 -> 4.1.1 (#384156)

adamcstephens committed:
garage: 1.0 and 0.9 are no longer broken on Darwin (#366373)

r-ryantm committed:
memray: 1.15.0 -> 1.16.0

NickCao committed:
superlu_dist: init at 9.1.0 (#386883)

nixpkgs-merge-bot[bot] committed:
karp: 0-unstable-2025-02-21 -> 0-unstable-2025-03-05 (#387706)

Co-authored-by: Bot-wxt1221 <3264117476@qq.com>

rhelmot committed:
libinotify-kqueue: 20180201 -> 20240724 (#387512)

NickCao committed:
netgen: init at 6.2.2501 (#387788)

r-ryantm committed:
meerk40t: 0.9.7010 -> 0.9.7020

r-ryantm committed:
above: 2.7 -> 2.8

JohnRTitor committed:
zed-editor: 0.176.2 -> 0.176.3

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

JohnRTitor committed:
zed-editor: use finalAttrs.finalPackage whenever possible

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

nixpkgs-merge-bot[bot] committed:
clever-tools: 3.11.0 -> 3.12.0 (#387770)

Co-authored-by: florian-sanders-cc <florian-sanders-cc@users.noreply.github.com>

drupol committed:
vscode-extensions.github.copilot{-,chat}: 1.27{5.0 -> 7.1411}, 0.24.2025021302 -> 0.25.202502 (#387746)

trofi committed:
mold: 2.36.0 -> 2.37.0

Changes: https://github.com/rui314/mold/releases/tag/v2.37.0

K900 committed:
kdePackages: Gear 24.12.2 -> 24.12.3

katexochen committed:
libdnf: 0.73.4 -> 0.74.0 (#387778)

mweinelt committed:
evcc: 0.200.6 -> 0.200.7 (#387671)

mweinelt committed:
meerk40t: 0.9.7010 -> 0.9.7020 (#387800)

jian-lin committed:
emacs: add a patch to fix upstream bug#63288 and bug#76523 (#387574)

JohnRTitor committed:
zed-editor.fhs: add entrypoint zed-editor-fhs in all-packages

Just like vscode-fhs. This will make it show up on `search.nixos.org`, and also cache it.

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

FliegendeWurst committed:
restinio: 0.7.3 -> 0.7.4 (#386835)

drupol committed:
vscode-extensions.github.copilot-chat: 0.25.2025021201 -> 0.26.2025030506

drupol committed:
vscode-extensions.github.copilot: 1.277.1411 -> 1.279.1416

drupol committed:
vscode-extensions.github.{copilot, copilot-chat}: bump (compatible with VSCode 1.98) (#387817)

leona-ya committed:
jetbrains: 2024.3 -> 2024.3.6 (#386114)

flokli committed:
nixos/movim: run nixfmt-rfc-style (#387633)

GaetanLepage committed:
python312Packages.pyiceberg: 0.8.1 -> 0.9.0

Diff: https://github.com/apache/iceberg-python/compare/refs/tags/pyiceberg-0.8.1...pyiceberg-0.9.0

Changelog: https://github.com/apache/iceberg-python/releases/tag/pyiceberg-0.9.0

oluceps committed:
go-graft, gg: merge duplicated package

oluceps committed:
go-graft: alias to gg

GaetanLepage committed:
python3Packages.lida: init at 0.0.14 (#383724)

mweinelt committed:
evcc: set mainProgram

mweinelt committed:
nixos/evcc: support passing secrets with envsubst

and apply some newer best practices.

GaetanLepage committed:
python312Packages.pymc: 5.21.0 -> 5.21.1

Diff: https://github.com/pymc-devs/pymc/compare/refs/tags/v5.21.0...v5.21.1

Changelog: https://github.com/pymc-devs/pymc/releases/tag/v5.21.1

oluceps committed:
go-graft: clarify license to agpl3Only

oluceps committed:
go-graft: add changelog

wegank committed:
xdg-desktop-portal-gtk: 1.15.1 -> 1.15.2 (#372545)

Scrumplex committed:
python312Packages.mandown: 1.11.1 -> 1.11.2 (#387782)

Scrumplex committed:
vencord: 1.11.5 -> 1.11.6 (#387708)

FliegendeWurst committed:
tracelinks: 1.0.1 -> 1.0.2 (#386943)

FliegendeWurst committed:
alire: 2.0.2 → 2.1.0 (#386989)

FliegendeWurst committed:
xan: 0.45.0 -> 0.46.2 (#386851)

happysalada committed:
aider-chat: 0.75.1 -> 0.75.2; add withPlaywright override (#387822)

GaetanLepage committed:
python312Packages.coiled: 1.79.2 -> 1.82.0 (#387649)

happysalada committed:
Windmill 1.410.3 -> 1.474.0 (#387629)

FliegendeWurst committed:
protoc-gen-swift: 1.28.2 -> 1.29.0 (#387103)

fabaff committed:
memray: 1.15.0 -> 1.16.0 (#387797)

fabaff committed:
above: 2.7 -> 2.8 (#387802)

Ma27 committed:
Merge: nixos/nextcloud: use LoadCredential to read secrets (#367433)

r-ryantm committed:
gancio: 1.23.1 -> 1.24.0

fabaff committed:
python313Packages.dissect-volume: disable failing test (#372357)

fabaff committed:
python313Packages.iterfzf: 1.4.0.60.2 -> 1.5.0.60.2

Diff: https://github.com/dahlia/iterfzf/compare/refs/tags/1.4.0.60.2...1.5.0.60.2

Changelog: https://github.com/dahlia/iterfzf/releases/tag/1.5.0.60.2

emilylange committed:
electron_32-bin: mark as insecure because it's EOL, electron-source.electron_32: remove as it's EOL (#383660)

fabaff committed:
python313Packages.iterfzf: 1.4.0.60.2 -> 1.5.0.60.2 (#387852)

SuperSandro2000 committed:
networkmanager: 1.50.2 -> 1.52.0 (#386037)

SuperSandro2000 committed:
decklink: fix build for kernel 6.13 (#387352)

mweinelt committed:
evcc: add envsubst secret support (#387829)

fpletz committed:
tracexec: 0.8.2 -> 0.10.0 (#386684)

K900 committed:
kdePackages: Gear 24.12.2 -> 24.12.3 (#387835)

Aleksanaa committed:
gnome-commander: 1.18.1-unstable-2024-10-18 -> 1.18.2 (#386966)

GaetanLepage committed:
aapt: disable on aarch64-linux

ddogfoodd committed:
typst: 0.13.0 -> 0.13.1

Diff: https://github.com/typst/typst/compare/v0.13.0...v0.13.1
Changelog:
https://github.com/typst/typst/blob/main/docs/changelog/0.13.1.md

Aleksanaa committed:
vimix-cursor-theme: remove (#387165)

Aleksanaa committed:
protoc-gen-connect-es: remove (#383159)

Aleksanaa committed:
lzwolf: drop (#386243)

drupol committed:
rabbit: fix build, relax dependencies

Aleksanaa committed:
kermit: fix build (#386337)

Aleksanaa committed:
cachix: set meta.mainProgram (#385825)

GaetanLepage committed:
aapt: disable on aarch64-linux (#387876)

r-ryantm committed:
renode-dts2repl: 0-unstable-2025-02-19 -> 0-unstable-2025-03-05

Aleksanaa committed:
swayosd: 0.1.0 -> 0.2.0 (#387052)

drupol committed:
rabbit: fix build, relax dependencies (#387879)

Aleksanaa committed:
cartridges: 2.11 -> 2.11.1 (#382939)

pbsds committed:
python313Packages.trimesh: 4.6.3 -> 4.6.4 (#386597)

pbsds committed:
castero: fix build (#386538)

pbsds committed:
python313Packages.anybadge: fix build, modernize (#386526)

pbsds committed:
python313Packages.mrsqm: r4 -> v.0.0.7 (#386524)

ddogfoodd committed:
typst: use finalAttrs

pbsds committed:
python313Packages.sopel: fix build (#386507)

pbsds committed:
python313Packages.datafusion: fix build (#386867)

azahi committed:
mold: 2.36.0 -> 2.37.0 (#387813)

drupol committed:
vscode-extensions.johnpapa.winteriscoming: init at 1.4.4 (#387716)

emilylange committed:
electron_32-bin,  electron-chromedriver_32: 32.3.2 -> 32.3.3 (#387264)

Aleksanaa committed:
go-graft, gg: merge duplicated package (#387824)

GaetanLepage committed:
zed-editor: 0.176.2 -> 0.176.3 (#387808)

Scrumplex committed:
libsForQt5.qtstyleplugin-kvantum: 1.1.3 -> 1.1.4 (#386896)

TomaSajt committed:
vyxal: update mill dependency to use fetchurl for version and URL locking (#387820)

nixpkgs-merge-bot[bot] committed:
gancio: 1.23.1 -> 1.24.0 (#387847)

Co-authored-by: jbgi <jbgi@users.noreply.github.com>

r-ryantm committed:
python312Packages.pgcli: 4.1.0 -> 4.2.0

Aleksanaa committed:
pacparser: add darwin support (#347951)

Aleksanaa committed:
scite: 5.5.4 -> 5.5.5 (#385027)

bobby285271 committed:
pantheon.appcenter: 8.0.1 -> 8.1.0

https://github.com/elementary/appcenter/compare/8.0.1...8.1.0

onny committed:
opensnitch-ui: 1.6.7 -> 1.6.8 (#387648)

Aleksanaa committed:
python3Packages.aria2p: Change python3Packages.appdirs to python3Packages.platformdirs in dependencies (#380746)

BenChand committed:
postgresqlPackages.pg_tle: init at 1.4.0 (#387868)

nixpkgs-merge-bot[bot] committed:
wit-bindgen: 0.39.0 -> 0.40.0 (#387792)

Co-authored-by: xrelkd <xrelkd@users.noreply.github.com>

GaetanLepage committed:
typst: 0.13.0 -> 0.13.1 (#387877)

donovanglover committed:
komac: migrate to `finalAttrs` (#387439)

roberth committed:
netlify-cli: 18.0.4 -> 18.1.0 (#384433)

SuperSandro2000 committed:
python312Packages.pgcli: 4.1.0 -> 4.2.0 (#387898)

Aleksanaa committed:
hurl: Use nix curl instead of bundled curl to enable http2 support (#387593)

GaetanLepage committed:
python312Packages.pymc: 5.21.0 -> 5.21.1 (#387831)

GaetanLepage committed:
python312Packages.pyiceberg: 0.8.1 -> 0.9.0 (#387827)

GaetanLepage committed:
zed-editor: gitUpdater -> nix-update-script

bobby285271 committed:
Pantheon updates 2024-03-07 (#387627)

nixpkgs-merge-bot[bot] committed:
renode-dts2repl: 0-unstable-2025-02-19 -> 0-unstable-2025-03-05 (#387883)

Co-authored-by: otavio <otavio@ossystems.com.br>

nh2 committed:
ceph: 19.2.0 -> 19.2.1 (#381075)
